signing-with-metamask/index.md

```diff
@@ -0,0 +1,89 @@
+
+Below are some of the critical milestones in the journey of signing data and transactions with MetaMask:
+
+2016: MetaMask was first released as a browser extension for Google Chrome, allowing users to interact with the Ethereum blockchain and sign transactions with their private keys.
+2017: MetaMask added support for signing data and messages, allowing users to prove ownership of an Ethereum address.
+2018: MetaMask introduced the ability to sign typed data, a specific format for structured data specified by the EIP-712 standard. This feature enabled users to sign more structured and organized data.
+2019: MetaMask introduced the ability to sign and send transactions directly from the browser extension, eliminating the need for users to copy and paste the transaction data into a separate signing tool.
+2020: MetaMask introduced the ability to sign transactions using hardware wallets, such as Trezor and Ledger, providing users with an additional layer of security.
+2021: MetaMask introduced the ability to sign messages using a new, more secure method called the "Ethereum Message Signing Protocol (EMSP)".
+
+MetaMask has continued to evolve and improve throughout its journey, providing users with more secure and user-friendly ways to sign data and transactions on the Ethereum blockchain. With the latest updates, MetaMask is now one of the most popular tools for signing data and transactions on the Ethereum network, used by developers and users alike.
+
+Using ethers.js for signing data and transactions on the Ethereum blockchain with MetaMask is convenient, providing an alternative approach. It offers a developer-friendly API and better support for TypeScript, which makes it easier for developers to build dApps and interact with the Ethereum blockchain.
+
+While MetaMask can provide a user-friendly interface for signing data and transactions, ethers.js, an alternative to web3.js (another JavaScript library for interacting with the Ethereum blockchain), has been focused on providing a developer-friendly library for building dApps and interacting with the Ethereum blockchain. Developers have widely adopted ethers.js as a more intuitive and easy-to-use library for signing data and transactions.
+
+MetaMask and ethers.js are not mutually exclusive and can be used together; developers can use MetaMask to interact with the Ethereum blockchain and sign data and transactions, and use ethers.js to build their dApp's front-end. ethers.js may also be used to sign data and transactions on other Ethereum clients such as Geth or Parity.
+
+## Methods of Signing Data With MetaMask
+
+We'll explore various ways of signing data using the provider injected by MetaMask as well as implementations in web3.js and ethers.js:
+
+eth_sign: This method is used to sign data with the Ethereum address's private key and returns the signature in the format of `"0x<r><s><v>"`.
```

Comment:
It would be better to de-emphasize our `eth_sign` method as much as possible. It has been deprecated for years, and we're considering removing it by default in an upcoming release. It poses a substantial risk to users. If we're to mention it at all, it should be last.